src/Components/App/App.js

```diff
@@ -31,28 +34,43 @@ class App extends React.Component {
   fetchMovies = () => {
```

Comment:
Adding an APICalls file might help with your code organization - not needed but something to consider!